### PR TITLE
Adds a 1s delay between START commands

### PR DESCRIPTION
**Pull request by darrylmasson**
Currently the boards enter the READY state when that reader finishes its arming sequence. If one host takes longer to arm then boards on the other hosts sit in this state for a while. Normally this isn't an issue but we've seen situations where these boards sometimes briefly flare to life for unknown reasons, which wrecks the rollover logic for that run. This PR changes how the end of the arming sequence behaves. Boards are now set to "idle" after arming, only being placed in a readied state on the receipt of the START command. The crate controller now receives the START command one second after the readers to allow for enough time for all boards to enter READY but hopefully limiting time for potential S-IN noise to cause shenanigans.

### CODE REVIEW

**Comment by darrylmasson** [comment]
Doesn't seem to cause any wonkiness.

File: dispatcher/DAQController.py [hunk 1]
```diff
@@ -274,16 +274,22 @@ def ControlDetector(self, command, detector, force=False):
...
         if (dt > self.timeouts[command] and dt_last > self.time_between_commands) or force:
             run_mode = self.goal_state[detector]['mode']
-            if command in ['start','arm']:
+            if command == 'arm':
                 readers, cc = self.mongo.GetHostsForMode(run_mode)
+                hosts = (cc, readers)
                 delay = 0
+            elif command == 'start':
+                readers, cc = self.mongo.GetHostsForMode(run_mode)
+                hosts = (readers, cc) # we want the cc to delay by 1s
```
Comment (dipedeserena) [inline dispatcher/DAQController.py:283]: I believe there is a reason, but I really cannot find it: why did you change the order of the hosts here?

Comment (darrylmasson) [reply dispatcher/DAQController.py:283]: If we specify a delay between when hosts get this command, (https://github.com/AxFoundation/redax/blob/master/dispatcher/MongoConnect.py#L420), then whichever hosts are listed second receive the command second. For START commands we want the readers to get the command before the CC, and for STOP it's the other way around.

Comment (dipedeserena) [reply dispatcher/DAQController.py:283]: ok, thanks
